docs/docs/queries/structure.md

```diff
@@ -1,44 +1,207 @@
 # Structure of a Query
 
-The dataview query language is a simple, structured, custom query language for quickly creating views on your data. It
+Dataview provides [multiple ways](dql-js-inline.md) to create queries and the way you need to write a query differs for each.
+
+This page provides information on how to write a **Dataview Query Language** (**DQL**) query. If you're interested in how to write Inline Queries, refer to the [inline section on DQL, JS and Inlines](dql-js-inline.md#inline-dql). You'll find more information about **Javascript Queries** on the [Javascript Reference](../api/intro.md).
+
+**DQL** is a SQL like query language for creating different views or calculations on your data. It
```

Comment:
Can we link to something about SQL? I imagine a good chunk of users haven't heard of it.

Reply:
I would like to see Wikiversity. It was a decent intro.

Reply:
You mean this? https://en.wikiversity.org/wiki/SQL
this seems rather ... short. Honestly, I'd just leave it, people can search if they're really interested. And it doesn't help learning SQL in order to write dataview, in my opinion - better focus on the real thing right from the beginning, so there's no understanding-benefit either.

Reply:
Yeah sounds fair. Let's maybe leave a link in the FAQ or additional resources for the curious reader instead.